internal/examples/thrift-hello/hello/main.go

```diff
@@ -37,60 +36,52 @@ import (
 //go:generate thriftrw --plugin=yarpc echo.thrift
 
 func main() {
-	httpTransport := http.NewTransport()
+	// configure a dispatcher for the service "hello",
+	// expose the service over an HTTP inbound on port 8086,
+	// and configure outbound calls to service "hello" over HTTP port 8086 as well
 	dispatcher := yarpc.NewDispatcher(yarpc.Config{
 		Name: "hello",
 		Inbounds: yarpc.Inbounds{
-			httpTransport.NewInbound(":8086"),
+			http.NewTransport().NewInbound(":8086"),
 		},
 		Outbounds: yarpc.Outbounds{
 			"hello": {
-				Unary: httpTransport.NewSingleOutbound("http://127.0.0.1:8086"),
+				Unary: http.NewTransport().NewSingleOutbound("http://127.0.0.1:8086"),
 			},
 		},
 	})
 
+	// register the Thrift handler which implements echo.thrift,
+	// whose shapes are available in the hello/ dir, as generated by
+	// the go:generate statement at the top of this file
 	dispatcher.Register(helloserver.New(&helloHandler{}))
-	client := helloclient.New(dispatcher.ClientConfig("hello"))
 
+	// start the dispatcher, which enables requests to be sent and received
 	if err := dispatcher.Start(); err != nil {
 		log.Fatal(err)
 	}
 	defer dispatcher.Stop()
```

Comment:
That would be awesome - hopefully only adding ~10 lines to the end of main...? cc @kriskowal can you update this example to do that?